<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
/*border-image:url(:/imagen/fondoMalv2.jpg);*/
background-repeat : no-repeat;
border:5px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="cargaManualContainer" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>591</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="cargaManualLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="tamBarcoLabel_2">
          <property name="text">
           <string>BARCO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nomBarcoLabel">
          <property name="text">
           <string>TAM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="posXLineEdit_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="posYLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="agregarButton">
        <property name="text">
         <string>AGREGAR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="dispararlLayout">
     <item>
      <widget class="QLineEdit" name="posXDisp"/>
     </item>
     <item>
      <widget class="QLineEdit" name="posYDisp"/>
     </item>
     <item>
      <widget class="QPushButton" name="dispararButton">
       <property name="text">
        <string>DISPARAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
